.github/workflows/push.yml

```diff
@@ -28,3 +28,12 @@ jobs:
 
     - name: Run tests
       run: make test
+
+    - name: Checks for new endpoints against AWS WAF rules
+      uses: cds-snc/notification-utils/actions/waffles@v1
+      with:
+        app-loc: '${{ github.workspace }}'
+        app-libs: '/doc-api-env/Lib/site-packages'
```

Comment:
I probably  need to change that to wherever the distribution will store the `site-packages` folder.

Reply:
I'll take a guess with `/usr/local/lib/python3.9/site-packages`..